packages/chargebee/src/action/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import Script from 'react-load-script';
+import { useOnUnmount } from './hooks';
+
+const Action = ({
+  site,
+  onClick,
+  onCall,
+  ...rest
+}) => {
+  const instanceRef = useRef();
+  const [isReady, setReady] = useState();
+
+  useOnUnmount(() => {
+    if (instanceRef.current) instanceRef.current.logout();
+  });
+
+  const handleLoad = () => {
+    try {
+      // Try to get chargebee instance.
+      // getInstance method throws an error if instance was not created
+      instanceRef.current = global.Chargebee.getInstance();
+    } catch (e) {
+      instanceRef.current = global.Chargebee.init({ site });
+    }
+
+    setReady(true);
+  };
+
+  let node;
+  if (isReady) {
+    const handleClick = (event) => {
+      if (onClick) onClick(event);
+      if (onCall) onCall(instanceRef.current, global.Chargebee);
+    };
+
+    node = <span {...rest} onClick={handleClick} />;
+  }
+
+  return (
+    <>
+      <Script url="https://js.chargebee.com/v2/chargebee.js" onLoad={handleLoad} />
```

Comment:
check for telemetry this script leaves behind. if possible destroy on unmount

Reply:
chargebee doesn't set any cookies.
it only sends error reports to https://nebenan-test.chargebee.com/api/internal/track_info_error

Reply:
any network activity/left over iframes after unmount?